ament_cmake_pep257/doc/index.rst

```diff
@@ -0,0 +1,42 @@
+ament_pep257
+============
+
+Checks the code style of Python source files using `pep257
+<http://pep257.readthedocs.org/>`_.
+Files with the following extensions are being considered: ``.py``.
+
+
+How to run the check from the command line?
+-------------------------------------------
+
+The command line tool is provided by the package `ament_pep257
+<https://github.com/ament/ament_lint>`_.
+
+
+How to run the check from within a CMake ament package as part of the tests?
+----------------------------------------------------------------------------
+
+``package.xml``:
+
+.. code:: xml
+
+    <build_depend>ament_cmake_test</build_depend>
```

Comment:
`ament_cmake_test` defines the CMake option `ament_tools` is setting: https://github.com/ament/ament_cmake/blob/8a2a607e82b2f1d89c24e63e52e81b3f8095e297/ament_cmake_test/ament_cmake_test-extras.cmake#L18
WIthout that there would be no tests in CMake: https://github.com/ament/ament_cmake/blob/8a2a607e82b2f1d89c24e63e52e81b3f8095e297/ament_cmake_test/ament_cmake_test-extras.cmake#L26

`buildtool_depend` is actually more accurate. But the difference between that and `build_depend` is only relavant for cross compilation.